cli/upload.py

```diff
@@ -30,7 +30,7 @@ def upload_assay(assay_type: str, xlsx_path: str):
     # Read the .xlsx file and make the API call
     # that initiates the upload job and grants object-level GCS access.
     with open(xlsx_path, 'rb') as xlsx_file:
-        upload_info = api.initiate_upload(assay_type, xlsx_file)
+        upload_info = api.initiate_assay_upload(assay_type, xlsx_file)
 
     try:
         # Log in to gcloud (required for gsutil to work)
```

Comment:
I'd say it's better to separate `gcloud.login` errors from actual upload `_gsutil_assay_upload` errors.